src/views/FarmAuction/components/AuctionHistory.tsx

```diff
@@ -112,6 +110,8 @@ const AuctionHistory: React.FC<AuctionHistoryProps> = ({ mostRecentClosedAuction
             <Input
               disabled={!mostRecentClosedAuctionId}
               type="input"
+              inputMode="numeric"
+              pattern="^[0-9]+$"
```

Comment:
we have use native input validation for max
```suggestion
              type="number"
              max={mostRecentClosedAuctionId}
              min={1}
```

Reply:
need to change `historyAuctionId` to type number in this way

Reply:
If type number is used, it doesn't show numpad in mobile browsers (on safari in particular), thats why i use type text with numeric input mode

Reply:
It's first time I heard about this restriction 🤔 you can do your way

Reply:
When type="number" used

![image](https://user-images.githubusercontent.com/2213635/143226646-e42702b8-90d5-4ddb-9eda-51770f812439.png)

when inputMode="numeric" used 

![image](https://user-images.githubusercontent.com/2213635/143226736-7aaf6ec5-d95a-4899-9ee0-e07c077f65bd.png)

Reply:
@memoyil what if we use type number and inputMode numeric? ios still shows text stuff?
To be honest as long as it works well on mobile I don't really care if its text type 🤷

Reply:
You can't use both, it should be either type=text and numeric or type=number